.github/workflows/ci-cd.yaml

```
@@ -24,6 +24,14 @@ jobs:
         run: npm run test
       - name: Upload coverage data
         uses: codecov/codecov-action@v1
+      - name: SonarCloud Scan
+        uses: SonarSource/sonarcloud-github-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Needed to get PR information, if any
+        with:
+          args: >
+            -Dsonar.javascript.lcov.reportPaths=./coverage/lcov.info
+            -Dsonar.projectVersion=${{github.ref_name}}
```

According to https://github.com/alma/main/blob/8b2cabecc8bce37e1e81ab317ee12228a2fcd6d5/.github/workflows/code_quality.yml#L69 you also need the `SONAR_TOKEN` secret

```suggestion
      - name: SonarCloud Scan
        uses: SonarSource/sonarcloud-github-action@master
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Needed to get PR information, if any
          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
        with:
          args: >
            -Dsonar.javascript.lcov.reportPaths=./coverage/lcov.info
            -Dsonar.projectVersion=${{github.ref_name}}
```

---

Yeah I tried to remove this line, this morning to see what would happen.

It's working now I have another error 'sonar.projectKey, sonar.organization' are mandatory, I'll add those and it should be good. Thanks